packages/@okta/vuepress-site/code/rest/index.md

```diff
@@ -21,19 +21,26 @@ You need a free Okta developer edition org to get started. Don't have one? [Crea
 
 2. [Install the Postman app](https://www.getpostman.com/apps).
-3. Launch Postman and select **Import** from the **File** menu.
-4. Click **Import From Link** and then paste this link into the box that appears: `https://developer.okta.com/docs/api/postman/example.oktapreview.com.environment`
-5. Click **Import**.
-6. After the import is complete, verify that the `{yourOktaDomain}` environment is selected in the upper-right corner of the page.
-
-  ![Postman app with collections](/img/postman_example_start.png "Points to the box in the upper-right corner that contains environments for use with Postman")
-
-7. Click the gear icon to the right of `{yourOktaDomain}` and select your org to replace or add these values:
-    * Rename your environment to something that you recognize, for example: `My Org`.
-    * `url`: Replace the example value with your org's full URL: For example, `https://mytestorg.oktapreview.com`. Make sure that you don't include `-admin` in the subdomain.
-    * `apikey`: Enter the API token that you created earlier, for example: `00LzMWxMq_0sdErHy9Jf1sijEGexYZlsdGr9a4QjkS`.
-
-8. Click **Update** to save your changes and then close the **Manage Environments** window.
+3. Start Postman if it's not open already.
+4. At the upper-left corner, click **Import**.
+![Postman window](/img/postman_import_button.png "Displays an arrow pointing to the Import button at the upper-left corner of the Postman window")
+5. In the **Import** dialog box, click **Link**, then paste this link: `https://developer.okta.com/docs/api/postman/example.oktapreview.com.environment` in the **Enter a URL** box and click **Continue**.
+![Postman Import dialog](/img/postman_import_dialog.png "Displays an arrow pointing to the Link tab in the Import dialog box")
+6. Under **Confirm your import**, click **Import**. ![Postman Import dialog for confirming import](/img/postman_import_dialog_confirm.png "Displays an arrow pointing to the Import button in the lower-right corner of the Import dialog box")
+A notification confirms that the import is complete.
+7. At the upper-right corner, click the box that lists the environments, and then select `{yourOktaDomain}` from the list. (The initial value is **No Environment** if you just installed Postman.) ![Postman environment list box](/img/postman_environment_list.png "Displays an arrow pointing to the box in the upper-right corner of the window that contains environments for use with Postman")
+8. At the upper-right corner, next to `{yourOktaDomain}`, click the **eye** symbol (Environment quick look). ![Postman environment quick look button](/img/postman_environment_quick_look.png "Displays an arrow pointing to the eye button in the upper-right corner of the Postman window")
```

Comment:
```suggestion
8. In the upper-right corner, next to `{yourOktaDomain}`, click **Environment quick look** (add an icon image here). ![Postman environment quick look button](/img/postman_environment_quick_look.png "Displays an arrow pointing to the eye button in the upper-right corner of the Postman window")
```

Style guide suggests using this format. Remove the large screen shot and replace it with a much smaller icon inline with the text that describes it. See Row 11 in the style guide here: https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/722929640/InfoDev+Style+Guide#InfoDevStyleGuide-ISS